sa/_db-next/migrations/20191108171706_RemoveOldAuthzTables.sql

```diff
@@ -0,0 +1,41 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+DROP TABLE authz;
+DROP TABLE pendingAuthorizations;
+DROP TABLE orderToAuthz;
+
+-- +goose Down
+-- SQL section 'Down' is executed when this migration is rolled back
+
+CREATE TABLE `authz` (
+  `id` varchar(255) NOT NULL,
+  `identifier` varchar(255) NOT NULL,
+  `registrationID` bigint(20) NOT NULL,
+  `status` varchar(255) NOT NULL,
+  `expires` datetime DEFAULT NULL,
+  `combinations` varchar(255) NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `registrationID_identifier_status_expires_authz_idx` (`registrationID`,`identifier`,`status`,`expires`),
+  CONSTRAINT `regId_authz` FOREIGN KEY (`registrationID`) REFERENCES `registrations` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+CREATE TABLE `pendingAuthorizations` (
```

Comment:
minor nit: I think these big `CREATE TABLE` statements in the down direction of the migration would benefit from a `\n` separator between them.